Assets/Scripts/CameraController.cs

```diff
@@ -71,15 +71,12 @@ private void Update()
 
     private void RotateKeyboardCamera()
     {
-        float rotateDir = (_targetY - _originalY)/90;
-        float moveAmount = rotateDir * Time.deltaTime * _speed;
-        _movedAmount += moveAmount;
-        if (_movedAmount > 90)
-            moveAmount -= _movedAmount - 90;
-        else if (_movedAmount < -90)
-            moveAmount -= _movedAmount + 90;
+        float moveAmount = Input.GetAxis("CameraControl") * Time.deltaTime * _keyboardSpeed;
         transform.RotateAround(PlayerObject.transform.position, Vector3.up, moveAmount);
-        if (Mathf.Abs(_movedAmount) >= 90f)
+
+        PlayerObject.GetComponent<PlayerController>().UpdateCamera();
```

Comment:
Could we cache the player in the `void Start()` method. `GetComponent` is rather expensive.

Reply:
sounds good